csv2bufr/resources/mappings/bufr_to_json.json

```diff
@@ -0,0 +1,241 @@
+{
+  "type": "Feature",
```

Comment:
We should add the `id` property before the `type` property, with a unique identifier.